notebooks/community/ml_ops/stage7/get_started_with_model_monitoring_xgboost.ipynb

```diff
@@ -0,0 +1,1477 @@
+{
```

Comment:
<p>Does it mean it runs the monitoring job every 1 hour? Can you please clarify that in one sentence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>done</p>